tests/integration/test_fim/test_files/test_file_limit/test_file_limit_capacity_alerts.py

```diff
@@ -72,6 +72,7 @@
 from wazuh_testing.tools import PREFIX
 from wazuh_testing.tools.configuration import load_wazuh_configurations
 from wazuh_testing.tools.monitoring import FileMonitor, generate_monitoring_callback
+from wazuh_testing.modules.fim import FIM_DEFAULT_LOCAL_INTERNAL_OPTIONS as local_internal_options
```

Comment:
Where do you use it? I can't find it.

Reply:
I see that between the 52-55 lines, the _scheduled_ mode does not appear. Which is the reason?

Reply:
The local_internal_options variable is called by the configure_local_internal_options_module fixture, so it is not actually called in the test. The fixture detects if the variable is present and uses it.

Added the scheduled description.